### ISSUE TITLE
How do I change the owner of a document?

### ISSUE THREAD

**Issue by test-OnlyOffice** [ONLYOFFICE/CommunityServer#315]
Good afternoon. There was a problem while testing the product before purchase and implementation. If the user is fired and blocked in AD, there will be no one to change the access rights to his files or delete them. How do I change the owner of files? There is an idea to change the values of create_by and modified_by in the files_file table - but we do not have enough information - are there any other dependencies in other tables?

In general, the functionality is very lacking in the ability to access all files from the administrator's side.

**Comment by Carazyda** [comment]
Hello @test-OnlyOffice you can reassign data for blocked users, please read this [tutorial](https://helpcenter.onlyoffice.com/userguides/groups-gettingstarted-people.aspx). In common documents you can change the owner of documents / folders through the context menu. Changing the database is not a good idea.

And please post on github issues if you have a problem, that something is not working. If you would like to know more about our functionality, you can read our [helpcenter](https://helpcenter.onlyoffice.com/index.aspx) or request a [demo](https://www.onlyoffice.com/demo-order.aspx) of what you need.

**Comment by rtoads** [comment]
I am running OnlyOffice CE Version 11.1.0.1506. Assigning documents to another user failed to work for me with two different disabled users and two different destination users. I received success messages and emails, but the documents did not get reassigned. When I finally tried reassigning with an immediate deletion of the user, the document owner became “User removed”.

We can't delete or move these documents. Is there a workaround?

**Comment by Carazyda** [comment]
Hello @rtoads Did you read this [tutorial](https://helpcenter.onlyoffice.com/userguides/groups-gettingstarted-people.aspx)? I want to draw your attention to the fact that only shared documents from the My Documents folder are transferred.

In the folder Common Documents - the administrator can change the owner of the document.

**Comment by rtoads** [comment]
Thanks for the reply. The documents in question were indeed in the My Documents folder and not the Common Documents. On the page you linked to, it says "Personal documents that are available to other portal users will be transferred to the My documents section of the selected active user where a new folder will be created with the name like 'Documents of user Merelin Linsdom'." But I do not see such a folder. Also, why do the documents get moved out of their current location when transferred?

**Comment by rtoads** [comment]
Also, strangely, this issue doesn't appear when searching for "is:issue is:open owner" in Gitgub.

**Comment by Carazyda** [comment]
Are you sure that the blocked user from whom you are transferring data had shared documents?

> Also, why do the documents get moved out of their current location when transferred?



Do not quite understand what you mean? Shared documents are transferred from one user to another.

**Comment by rtoads** [comment]
I am sure that the user had shared documents. By this I mean "Personal documents that are available to other portal users".  Is this what you mean? The documents are still shared and accessible by other users, but the owner of the document is now “User removed” after I attempted reassign these documents to another user during the user deletion process.

**Comment by rtoads** [comment]
I'm also wondering why the OnlyOffice permissions model doesn't really give Full Access to users and groups. In Unix, If you have R/W privileges, you can delete the file. In OnlyOffice, only the owner can delete, which creates all kinds of problems.

**Comment by Carazyda** [comment]
Sorry, but there is no way we can repeat your problem. Does the problem persist for another user? Can you write us a detailed script step by step?

**Comment by KlevankyA** [comment]
So, is there any way for a user who has full acess to the files delete them without being the owner?

**Comment by Carazyda** [comment]
> So, is there any way for a user who has full acess to the files delete them without being the owner?

No, in Common Documents only owners and administrators have the ability to delete files.

**Comment by VincentSC** [comment]
The problem is that there is no way to make a team or user the admin of a directory. So only site-wide admins can delete or move a file. 

When a team uses their own directory, and all its members enthusiastically help create and organize files, we found it quickly becomes a mess due to the unmoveable files. It's simply more efficient to ask the site-admin to help archive+clean a directory, each time when it's needed, than to coordinate it with the team. We found out that's at least once a month, and it's becoming a growing tension, for both site-admins and the teams.

IMO, this should be provided:
1. no access
2. read access (view)
3. write access (edit+create)
4. local admin (delete+move)

Just check the icons, and you see it's weird:
- owner = **star**. level 4
- admin = **pen**. level 4 (??)
- team = **pen**. level 3

Please fix this by:
- putting a star to "admins", and
- alllowing to use "admin, level 4" as an option
